docs/dapp/sapphire/README.mdx

```diff
@@ -13,90 +14,81 @@ Sapphire allows for:
-          blockExplorerUrls: ['https://explorer.sapphire.oasis.io'],
-        },
-      ],
-    })
-  }}
->
-  Click here to register Sapphire Mainnet to your MetaMask or Brave Wallet
-</button>
+- Network name: `sapphire`
+- Long network name: `Oasis Sapphire`
+- Chain ID:
+  - Hex: `0x5afe`
+  - Decimal: `23294`
 
 ### Testnet
 
-* RPC HTTP endpoint: `https://testnet.sapphire.oasis.dev`
-* RPC WebSockets endpoint: `wss://testnet.sapphire.oasis.dev/ws`
-* Chain ID:
-  * Hex: 0x5aff
-  * Decimal: 23295
-* Block explorer: [https://testnet.explorer.sapphire.oasis.dev](https://testnet.explorer.sapphire.oasis.dev)
-
-<button
-  class="button button--primary margin-bottom--md"
-  onClick={() => {
-    if (!window.ethereum?.request) {
-      return alert('Have you installed MetaMask yet? If not, please do so.\n\nComputer: Once it is installed, you will be able to add the ParaTime to your MetaMask.\n\nPhone: Open the website through your MetaMask Browser to add the ParaTime.')
-    }
-    window.ethereum.request({
-      method: 'wallet_addEthereumChain',
-      params: [
-        {
-          chainId: '0x5aff',
-          chainName: 'Oasis Sapphire Testnet',
-          nativeCurrency: {
-            name: 'TEST',
-            symbol: 'TEST',
-            decimals: 18,
-          },
-          rpcUrls: ['https://testnet.sapphire.oasis.dev/', 'wss://testnet.sapphire.oasis.dev/ws'],
-          blockExplorerUrls: ['https://testnet.explorer.sapphire.oasis.dev'],
-        },
-      ],
-    })
-  }}
->
-  Click here to register Sapphire Testnet to your MetaMask or Brave Wallet
-</button>
+- Network name: `sapphire-testnet`
+- Long network name: `Oasis Sapphire Testnet`
+- Chain ID:
+  - Hex: `0x5aff`
+  - Decimal: `23295`
+
+## RPC Endpoints
+
+:::danger
+
+The RPC endpoint is a *point of trust*. Beside traffic rate limiting, it can
+also perform censorship or even a man-in-the-middle attack. If you have security
+considerations, we strongly recommend that you set up your own [ParaTime client
+node][paratime-client-node] and the [Web3-compatible gateway].
+
+:::
+
+[Web3-compatible gateway]: ../../node/web3.md
+[paratime-client-node]: ../../node/run-your-node/paratime-client-node.mdx
+
+You can connect to one of the public Web3 gateways below (in alphabetic order):
+
+| Provider                  | Mainnet RPC URLs     | Testnet RPC URLs       | Supports Confidential Queries |
+|---------------------------|----------------------|------------------------|-------------------------------|
+| [1RPC] | <S rpcs={['https://1rpc.io/oasis/sapphire']} /> | N/A | Yes |
+| [Oasis Protocol Foundation] | <S rpcs={['https://sapphire.oasis.io','wss://sapphire.oasis.io/ws']} /> | <ST rpcs={['https://testnet.sapphire.oasis.dev','wss://testnet.sapphire.oasis.dev/ws']} /> | Yes |
+
+Public RPCs may have rate limits or traffic restrictions. For professional,
+dedicated RPC endpoints, consider the following providers (in alphabetic order):
+
+- [1RPC]
```

Comment:
Please, create a table for dedicated private RPC enpoints that has:
- link to docs on how to use it
- link to pricing info
- ...